### DISCUSSION TITLE
Red point position on map in drives module

### DISCUSSION THREAD

**Discussion by BigPapoo** [teslamate-org/teslamate#2854] [General]
Hi, am I wrong or the last version (1.27.1) is missing a feature that, in the past, was showing a big red point on the path (when a map is displayed) when moving the mouse back and forth on any chart? The point relates to the car position on the map for a given stat time. This would be very cool to have it back in the next release.

**Comment by cwanja** [comment]
#2770

**Reply by BigPapoo** [reply]
Got it! Thx.
